src/graphql/resolvers/abilityScoreResolver.ts

```diff
@@ -1,15 +1,22 @@
 import SkillModel from '../../models/skill/index.js';
 import { coalesceFilters, getMongoSortDirection, resolveContainsStringFilter } from './common.js';
+import { AbilityScore } from '../../models/abilityScore/types';
+
+type SortParams = Record<string, 1 | -1>;
```

Comment:
I think sort params are pretty much always a `Record<string, 1 | -1>`. You could put this type in `common.ts` and reuse it wherever you use sort.

Reply:
This required a slight refactor because TS was angry about `direction = -direction` in `coalesceSort`.